internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/fastclient/AvroStoreClientEndToEndTest.java

```diff
@@ -215,6 +218,171 @@ protected void runTest(
     }
   }
 
+  private void runTestWithClientException(
+      ClientConfig.ClientConfigBuilder clientConfigBuilder,
+      boolean batchGet,
+      int batchGetKeySize,
+      Consumer<MetricsRepository> statsValidation,
+      Optional<AvroGenericStoreClient> vsonThinClient) throws Exception {
+    MetricsRepository metricsRepositoryForGenericClient = new MetricsRepository();
+    AvroGenericStoreClient<String, GenericRecord> genericFastClient =
+        getGenericFastClient(clientConfigBuilder, metricsRepositoryForGenericClient, true);
+
+    AvroGenericStoreClient<String, Object> genericFastVsonClient = null;
+    // Construct a Vson store client
+    genericFastVsonClient =
+        getGenericFastVsonClient(clientConfigBuilder.clone(), new MetricsRepository(), vsonThinClient, true);
+    try {
+      if (batchGet) {
+        // test batch get of size 2 (current default max)
```

Comment:
I believe we do have stream batch get available that no longer have this limitation. Perhaps we could use that API instead since we are deprecating this old batch get API.

Reply:
Removed test for `batchGetKeySize==2` here, Manoj gave the okay

Reply:
Reused `runTest` and removed this method altogether